configs/sst_pt_gcc-misc.yaml

```diff
@@ -10,6 +10,10 @@ data:
     - binutils-gold
     - annobin-annocheck
     - annobin-plugin-gcc
+    # used by several sst_pt tools (libabigail, dwz, binutils, debugedit)
+    - xxhash
+    - xxhash-libs
+    - xxhash-devel
```

Comment:
fa5645be2dd1ccd543e286e8137654d36e8b6377 also had `xxhash-docs`, but I reckon we don't need it if we keep it as buildroot-only.

Reply:
There's been some back-and-forth on xxhash; last I heard, it was going to be buildroot-only in order to build these tools package, and header-only mode so that there is no runtime dependency on xxhash-libs.  Is that still the case?  Do you plan on shipping any part of xxhash in CRB (technically not necessary)?

Reply:
I suspect we'd like to remain open to the possibility of CRB, not just buildroot-only, since some packages may want to use it in a mode not-only-inline.  Is there a downside in keeping the options open?

Reply:
In that case I think it would suffice to delist xxhash as unwanted but not add it here.  If it gets to the point that xxhash-libs is pulled in as a dependency (by a package using it *not* in inline mode) then it would make sense to ship the rest in CRB.

Reply:
OK, whatever works.  (I'm curious what the consequences are of listing the packages here, closer to their new packager sst, vs. not listing explicitly anywhere).

Reply:
Lets start with delisting from unwanted and see where this goes.

Reply:
(By the way, how do we "see where this goes"?  The PRP docs talk about doing a "content resolver check", but don't describe how to check.)

Reply:
I was going to say that it shouldn't show up [here](https://tiny.distro.builders/view-unwanted--view-c10s.html) but it already doesn't...

Reply:
https://github.com/minimization/content-resolver-input/pull/1329   is the "un-unwanted" alternative.

Reply:
It won't show up in c10s CR until it is built again in c10s, so you can (and should) proceed with the PRP in the meantime.  Once built, xxhash(-devel) will show up again as a build dependency of at least dwz.

The question then remains if this will be buildroot only or will be shipped in CRB.

Reply:
This PR would be fine if/when you decide to add it to CRB.